tests/lib/store.sh

```diff
@@ -19,7 +19,9 @@ setup_fake_store(){
     # debugging
     systemctl status fakestore || true
     echo "Given a controlled store service is up"
-    systemd-run --unit fakestore $(which fakestore) -start -dir $top_dir -addr localhost:11028 $@
+    https_proxy=${https_proxy:-}
+    http_proxy=${http_proxy:-}
```

Comment:
Do we need those? I read it as "if it exists, defines to itself.. if it doesn't exist, define it to nothing", which sounds like the same as doing nothing.